lib/fluent/plugin/out_kafka_buffered.rb

```diff
@@ -74,6 +74,9 @@ class Fluent::KafkaOutputBuffered < Fluent::BufferedOutput
 requires activesupport gem - records will be generated under fluent_kafka_stats.**
 DESC
 
+  config_param :statsd_monitoring, :bool, :default => false
```

Comment:
How about using array type parameter instead of each single parameter like below?

```
config_param :monitoring_list, :array, :default => []
```

With this approach, we can write configuration simply.

```
@monitoring_list.each { |m|
  require "kafka/#{m}"
  log.info "..."
}
```